libs/template/spec/render-strategies/render-strategies/noop.strategy.spec.ts

```diff
@@ -0,0 +1,67 @@
+// tslint:disable-next-line:nx-enforce-module-boundaries
+import { mockConsole } from '@test-helpers';
+import { getStrategies } from '@rx-angular/template';
+import { getMockNoopStrategyConfig } from '../../fixtures';
+import { from, of } from 'rxjs';
+import createSpy = jasmine.createSpy;
+
+const noopStrategyName = 'noop';
+
+describe('noop Strategy', () => {
+  beforeAll(() => mockConsole());
+
+  it('should be present in strategies map', () => {
+    const strategy = getStrategies(getMockNoopStrategyConfig())[noopStrategyName];
```

Comment:
Can this be a file-level const and assignment placed in `beforeEach`? (applicable for all strategy specs files)